src/NHibernate/Type/NullableType.cs

```diff
@@ -332,11 +332,6 @@ public override bool[] ToColumnNullness(object value, IMapping mapping)
 			return value == null ? ArrayHelper.False : ArrayHelper.True;
 		}
 
-		public override bool IsEqual(object x, object y)
-		{
-			return Equals(x, y);
-		}
```

Comment:
This override has the exact same implementation as the overridden method, so removing. Moreover, overriding `IsEqual` without overriding `GetHashcode` is normally a serious bug. (But there, since the override was just redundant, it was not causing troubles.)